README.md

```diff
@@ -24,18 +106,91 @@ npm install -g npm@7.13.0
+
+- When you need to style a `::before` or `::after` pseudo element
+- When you need styles based on siblings (`+` or `~` selectors)
+- When you have styles that truly don't appear anywhere else in the designs (this should happen extremely rarely thought we've seen it most often with box-shadows)
+
+#### Storybook
+
+##### Naming
+
+If your component has a single Story, we use the default "Primary" naming convention.
+
+If your component has multiple stories, we name each story semantically to describe the use case.
+
+##### Use Decorators If You Need A Wrapper
+
+If you need a wrapper container around your story (for example, because you have multiple elements, or you need to apply styling to the container), use a decorator.
+
+See the [main navigation component story file](https://github.com/lob/ui-components/blob/main/src/components/MainNavigation/MainNavigation.stories.js#L79) for an example. 
+
+##### Documentation
+
+We're using `.mdx` files for documentation for each component. This allows us to use markdown to write more detailed docs but also integrate with Storybook's docs add-on blocks (e.g. things like automatically generating a preview of your component, the props table, etc.).
+
+At the minimum, each component's docs should have:
+
+- A preview section
+- An intro to the component - what it'd most commonly be used for
+- How to use the component, with code examples
+- A props section with the types and required fields complete
+
+##### Routing
+
+We've included Vue Router and a `routeDecorator` (`.storybook/routeDecorator.js`) so you can use dynamic routing in your components. If your component needs to be route-aware, either because it uses `router-link` or it checks `this.$route` in a computed prop or method, you'll need the route decorator in your Story.
+
+See the [main navigation](https://github.com/lob/ui-components/blob/main/src/components/MainNavigation/MainNavigation.stories.js#L14) and [subnavigation component story](https://github.com/lob/ui-components/blob/main/src/components/Subnavigation/Subnavigation.stories.js#L20) files for examples. 
+
+#### Accessibility
+
+While we're not strictly enforcing any particular WCAG standard (though we should strive for at least AA), we want to achieve a base-level of accessibility with our first iteration. This includes:
+
+- Keyboard navigable
+- Sensical for screen reader users
+- Color contrast passes AA at least
+
 ## Unit testing
 ```bash
 npm run test:unit
 ```
 
-### Lints and fixes files
+## Lints and fixes files
 ```bash
 npm run lint
 ```
 
+## Contributing
+### A Complete PR for a Component
+
+We consider a component PR complete when it has the following:
+
+- Storybook examples with the ability to trigger each state
+- Clear Storybook documentation
+- Base level of accessibility
+- Tests added/updated for any code added/updated
+- Existing tests & lint passes (integrated with CircleCI, but you can also run both locally)
 ## Publish to NPM
 ```bash
 npm run build-bundle
```

Comment:
I think this should actually be `npm run build-library`? (I know you didn't actually touch this, but might as well change it while we're here? 🙃 )